nvflare/fuel/flare_api/flare_api.py

```diff
@@ -300,7 +300,7 @@ def list_jobs(
                 command = command + " " + id_prefix
         result = self._do_command(command)
```

Comment:
we prob. should also update above as default is list all jobs (the same behavior as in 2.2)

Reply:
but this PR LGTM